src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardClaimMatcher.java

```diff
@@ -160,4 +163,75 @@ public boolean isHearingFormGenerated() {
         return !caseData.getHearingDocuments().isEmpty();
     }
 
+    @Override
+    public boolean sdoBeenDrawn() {
+        return false;
+    }
+
+    @Override
+    public boolean beforeHearing() {
+        return caseData.getHearingDate().isAfter(LocalDateTime.now().toLocalDate());
+    }
+
+    @Override
+    public boolean isMoreDetailsRequired() {
+        return sdoBeenDrawn() && beforeHearing();
+    }
+
+    @Override
+    public boolean isMediationSuccessful() {
+        return !sdoBeenDrawn()
+            && caseData.getMediation().getMediationSuccessful() != null;
+    }
+
+    @Override
+    public boolean isMediationUnsuccessful() {
+        return !sdoBeenDrawn()
+            && !caseData.getMediation().getUnsuccessfulMediationReason().isEmpty();
```

Comment:
Can we use `caseData.getMediation().getUnsuccessfulMediationReason()` instead double negation?